File: Linguagem C#/Terreno.cs
// See https://aka.ms/new-console-template for more information
using System.Globalization;

CultureInfo CI = CultureInfo.InvariantCulture;

Console.Write("Digite a largura do terreno: ");
double largura = double.Parse(Console.ReadLine(), CI);
Console.Write("Digite o comprimento do terreno: ");
double comprimento = double.Parse(Console.ReadLine(), CI);
Console.Write("Digite o valor do metro quadrado: ");
double valorMetroQuadrado = double.Parse(Console.ReadLine(), CI);
double area = largura * comprimento;
double preco = area * valorMetroQuadrado;
Console.WriteLine("Area do terreno = " + area.ToString("F2", CI));
Console.WriteLine("Preco do terreno = " + preco.ToString("F2", CI));

File: Linguagem C++/terreno.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {

    double valormetro,area,preco, largura, comprimento;

    cout << "Digite a largura do terreno:";
    cin >> largura;
    cout << "Digite o comprimento do terreno: ";
    cin >> comprimento;
    cout << "Digite o valor do metro quadrado: ";
    cin >> valormetro;

    area = largura * comprimento;
    preco = valormetro * area;
    cout << fixed << setprecision(2);
    cout << "Area do terreno = " << area << endl;
    cout << "Preco do terreno = " << preco << endl;



    return 0;
}
